tests/in/collatz.param.ron

```diff
@@ -4,7 +4,5 @@
 	spv_capabilities: [ Shader ],
 	spv_debug: true,
 	spv_adjust_coordinate_space: false,
-	mtl_bindings: {
-		(stage: Compute, group: 0, binding: 0): (buffer: Some(0), mutable: true),
-	}
+	msl_custom: false,
```

Comment:
nit: it could be useful to keep the `binding_map` instead of using the fake bindings for MSL validation when possible. e.g. just in case there are subtle interactions between valid bindings and the rest of the shader that won't appear when we use `user` attributes instead

Reply:
I'm trying to think of what those interactions could be, and unable to come up with anything. We can enable more tests to use the full `Options` struct as a follow-up?